training/css/training.css

```diff
@@ -0,0 +1,5 @@
+#table-test-login{
+    border: hidden;
+    background: rgba(102, 172, 255, 0.84);
+    text-align: center;
+}
```

Comment:
configure your IDE to put automatically a newline at the end of file or do it manually
see https://www.jetbrains.com/phpstorm/help/configuring-line-separators.html#d43786e126